src/common/MessageProcessor.sol

```diff
@@ -26,6 +26,8 @@ import {CastLib} from "../misc/libraries/CastLib.sol";
 import {BytesLib} from "../misc/libraries/BytesLib.sol";
 import {IRecoverable} from "../misc/interfaces/IRecoverable.sol";
 
+uint16 constant MAINNET_CENTRIFUGE_ID = 1;
```

Comment:
Any specific reason to put this outside instead of inside the contract?

Reply:
I think we use constants within the contract everywhere else now. Benefit is just that it is readable by other contracts.

Reply:
Good argument